test/views/EditRecordViewSpec.scala

```diff
@@ -60,9 +59,32 @@ class EditRecordViewSpec extends BaseSpec {
       LegalStatus("ref.4", "Welsh Public Record(s)")
     )
 
-  val emptyRecord: EditSetRecord = EditSetRecord.apply(
-    ccr = "",
-    oci = "",
+  val editSetRecord = EditSetRecord(
+    ccr = "COAL 80/80/1",
+    oci = "COAL.2022.V5RJW.P",
+    scopeAndContent = "",
```

Comment:
for consistency the oci needs to match with the ccr above

Reply:
It appears to match the record in `package` ... or perhaps I misunderstood.

Reply:
It should match the record in the package file. I changed it to match after the first comment.